portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/SegmentsOverviewAttributeBreakdown.testcase

```diff
@@ -36,8 +36,12 @@ definition {
 	}
 
 	@description = "Feature ID: Legacy | Automation ID: LRAC-8504 | Test Summary: Segment Overview distribution filtered by boolean"
+	@ignore = "true"
 	@priority = "3"
 	test SegmentOverviewDistributionFilteredByBoolean {
+
+		// CSV data source tests are no longer possible to run due to this ticket LRAC-11651
+
```

Comment:
Hey @manuferreira, I think this test can create a boolean type custom field to be used in place of the attribute that use the value from the CSV.

I think you can undo this change and create a ticket for us to refactor this test or you can do it yourself in your ticket